api/src/main/resources/org/openmrs/api/db/hibernate/Diagnosis.hbm.xml

```diff
@@ -54,10 +57,10 @@
 		
 		<many-to-one name="changedBy" class="org.openmrs.User" column="changed_by" />
 
-		<many-to-one name="encounter" class="org.openmrs.Encounter"
+		<many-to-one name="encounter" class="org.openmrs.Encounter" column="encounter_id"
 					 not-null="true" />
 
-		<many-to-one name="condition" class="org.openmrs.Condition"/>
+		<many-to-one name="condition" class="org.openmrs.Condition" column="condition_id"/>
 
```

Comment:
I see you've update the hbm file but I don't see the liquibase changesets to update the table in the database, are they already added?

Reply:
Yes, they are already added